js/scenes/NumberOfGuestScreen/NumberOfGuestScreen.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import {
+  Text,
+  View,
+  TouchableHighlight,
+  TextInput
+} from 'react-native'
+
+
+import styles from './styles'
+
+const NumberOfGuestScreen = ({ text, onTextChange, onIncrease, onDecrease }) => {
+  return (
+    <View>
+      <View style={styles.container}>
+        <TextInput
+          keyboardType={'numeric'}
+          style={styles.textInput}
+          value={text.toString()}
+          onChangeText={(text) => {
+            (onTextChange(text))
+          }} />
+          <View>
+          
+          </View>
+        <View style={styles.buttonContainer}>
+          <TouchableHighlight
+            style={[styles.button, styles.inactive]}
+            onPress={() => (onIncrease())}
```

Comment:
Function name is weird here. On press, you increase, right? So it should be:

```js
onPress={() => (increase())}
```

When you name something on[Action] the function should be an event handler.